views/movies/new-movie.hbs

```diff
@@ -0,0 +1,22 @@
+<h2>Create New Movie</h2>
+<form action='/movies/create/' method='post'>
+  <label>
+    Title:
+    <input type='text' name='title' />
+  </label>
+  <label>
+    Genre:
+    <input type='text' name='genre' />
+  </label>
+  <label>
+    Plot:
+    <input type='text' name='plot' />
+  </label>
+  <label>Cast:</label>
+  <select name='name'>
```

Comment:
By adding "multiple" you get to select more than one celebrity

`<select name='name' multiple>`